RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc

```diff
@@ -92,13 +95,13 @@ void SiPixelDigisClustersFromSoAT<TrackerTraits>::produce(edm::StreamID,
   for (uint32_t i = 0; i < nDigis; i++) {
     // check for uninitialized digis
     // this is set in RawToDigi_kernel in SiPixelRawToClusterGPUKernel.cu
```

Comment:
you can use
```c++
auto& digi = digis.view()[i];
```
and later use `digi.rawIdArr()` instead of `digis.view()[i].rawIdArr()`, _etc._